rclcpp/test/rclcpp/test_add_callback_groups_to_executor.cpp

```diff
@@ -191,17 +191,17 @@ TYPED_TEST(TestAddCallbackGroupsToExecutor, add_callback_groups_after_add_node_t
   rclcpp::CallbackGroup::SharedPtr cb_grp = node->create_callback_group(
     rclcpp::CallbackGroupType::MutuallyExclusive);
   rclcpp::TimerBase::SharedPtr timer_ = node->create_wall_timer(
-    2s, timer_callback, cb_grp);
+    10ms, timer_callback, cb_grp);
```

Comment:
why this change?

Reply:
This test takes 4 seconds for each executor (16 seconds when testing 4 different executors) due the period of the timer, which doesn't have any impact on the output of the test.

Reply:
I don't know why they were set to this before hand, but it might make the tests flaky. It really shouldn't, but I'm just throwing it out there, because it does happen if the test isn't constructed properly.